lua/autorun/server/sv_cfc_contact_forms.lua

```diff
@@ -145,6 +145,8 @@ local function sendFailureAlert( ply )
     net.Send( ply )
 end
 
+util.AddNetworkString( "Admin_alert" )
```

Comment:
This is a very generic net message, make sure to make it more unique. If another addon would use the same net message name stuff can conflict and will definitely cause errors. I suggest using something like `cfc_contact_forms_alert`.

Reply:
Changed net message name to "cfc_contact_forms_alert"